scopes/workspace/workspace/pattern.cmd.ts

```diff
@@ -9,9 +9,9 @@ export class PatternCommand implements Command {
   extendedDescription = `this command helps validating a pattern before using it in other commands.
 a pattern can be a simple component-id or component-name. e.g. "ui/button".
 a pattern can be used with wildcards for multiple component ids, e.g. "org.scope/utils/**" or "**/utils/**" to capture all org/scopes.
-to enter multiple patterns, separate them by a comma, e.g. "ui/*, lib/*"
-to exclude, use "!". e.g. "ui/**, !ui/button"
-always wrap the pattern with quotes to avoid collision with shell commands.
+to enter multiple patterns, separate them by a comma, e.g. 'ui/*, lib/*'
+to exclude, use "!". e.g. 'ui/**, !ui/button'
```

Comment:
This here aroun the! Should be ' not "

Reply:
can see you made the changes, i added some more in the examples so that it's all single '